common/persistence/dataInterfaces.go

```diff
@@ -152,6 +152,13 @@ const (
 	TransferTaskTypeUpsertWorkflowSearchAttributes
 )
 
+// Types of cross-cluster tasks
+const (
+	CrossClusterTaskTypeStartChildExecution = iota
```

Comment:
nit: it's good to have something like `CrossClusterTaskTypeInvalid` as default value, so that when we forget to set the value at some place, we'll get an explicit error.

Reply:
Good point. I changed the enum value to start from 1 so that if the value is unset the value will be an invalid value 0.